src/index.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './components/App';
+import 'bootstrap/dist/css/bootstrap.css';
+import * as actions from './actions/book';
+import store from './reducers/index';
+// import configureStoreApp from './reducers/configureStore';
+
+// const store = configureStoreApp();
+
+store.dispatch(actions.bookAdded('book 1', 'description of book 1', 'Horror'));
+store.dispatch(actions.bookAdded('book 2', 'description of book 2', 'Action'));
+store.dispatch(actions.bookAdded('book 3', 'description of book 3', 'Kids'));
```

Comment:
- [ ] <b>OPTIONAL:</b> This is not the best or rather the correct way to initialize the default state in redux. since this is a new change, It wasn't pointed out by the previous reviewer. so I will mark it as `optional` But I strongly recommend to go find the best way to initialize the default state in redux.